website/docs/events/m3-slides/.slides/evaluation.md

```diff
@@ -4,12 +4,71 @@
 
 ---
 
+## Evaluation
+
+- Student feedback
+- Lab Assistant Feedback
+- Grades
+- Contributions
+
+---
+
+## Student Evaluation
```

Comment:
Maybe it's not to our advantage, but I think you should mention the amount of students who submitted responses to the feedback form. All percentagee are divisible by 5, which makes them look kibda sus.

Reply:
If someone asks, I'll tell.